devutils/pyproject.toml

```diff
@@ -19,9 +20,7 @@ pytest-regressions = "1.0.6"
 pydocstyle = "^6.3.0"
 isort = { version = "^5.12.0", python = "^3.8" }
 types-invoke = "^2.0"
-invoke = "^2.0"
 lazydocs = { git = "https://github.com/robocorp/lazydocs.git" }
```

Comment:
I'm not sure I follow, what do we gain by removing things from pyproject.toml?

Reply:
Was thinking that this might take away from the confusion of who is responsible for providing you with the Invoke tool. Is it yourself initially or the Poetry-based venv after a `poetry install`? (this way you'd be sure that you should get Invoke from the _requirements.txt_)

Also it helps with pinning & maintaining a compatible version in a single place, which is _requirements.txt_. But if you'll need to use Invoke from Poetry's context Invoke again (e.g.: `inv <task>` -> `poetry(..., "inv", ...)`), then yes, it makes sense to keep that here.

Will add it back if it doesn't produce any harm or confusion.

Reply:
> What do we gain by adding unnecessary dependencies in devtools' pyproject.toml?

Nothing, but as you mentioned before:

> But if you'll need to use Invoke from Poetry's context (e.g.: inv <task> -> poetry(..., "inv", ...)), then yes, it might make sense to keep that dep here if the initial executable isn't reachable anymore.

So, we do get something here (although our automations using `inv` make it so that it doesn't need the poetry env active, sometimes I do expect `inv` or `poetry` to be in the env if I'm doing things that are not initially in our automations).